### PR TITLE
Pr/action admin

### PR DESCRIPTION
**Pull request by MethodenMann**

### CODE REVIEW

File: packages/app-extensions/src/actions/modules/actionHandlers/customAction.js [hunk 1]
```diff
@@ -1,8 +1,35 @@
...
+          () => <ActionComponent appId={definition.appId} selection={selection}/>,
+          true
+        ))
+      } else {
+        consoleLogger.logError('Unable to render custom action: No appComponent provided')
+      }
+    }
+  } else {
+    const handler = config && config.customActions && config.customActions[definition.id]
+    if (handler) {
```
Comment (rzueger) [inline packages/app-extensions/src/actions/modules/actionHandlers/customAction.js:31]: don't log error if no handler found?

File: packages/admin/src/routes/entities/components/ActionView/ActionView.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    return () => {
+      setCurrentViewTitle(null)
+    }
+  }, [currentViewInfo])
+
+  if (!currentViewInfo) {
+    return null
+  }
+
+  // TODO: Action better query param handler. allow tql
```
Comment (rzueger) [inline packages/admin/src/routes/entities/components/ActionView/ActionView.js:24]: what about this todo? left here intentionally?

File: packages/admin/src/routes/entities/components/ActionView/ActionView.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  if (!selection) {
+    return 'invalid selection'
+  }
+
+  return <Action appId={currentViewInfo.actionId} selection={selection}/>
+}
+
+ActionView.propTypes = {
+  intl: intlShape,
+  currentViewInfo: PropTypes.object,
```
Comment (rzueger) [inline packages/admin/src/routes/entities/components/ActionView/ActionView.js:34]: maybe define shape for currentViewInfo?

File: packages/admin/src/routes/entities/components/ActionView/ActionView.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  const selection = (location.state && location.state.selection) || querySelection
+  if (!selection) {
+    return 'invalid selection'
+  }
+
+  return <Action appId={currentViewInfo.actionId} selection={selection}/>
+}
+
+ActionView.propTypes = {
+  intl: intlShape,
```
Comment (rzueger) [inline packages/admin/src/routes/entities/components/ActionView/ActionView.js:33]: both `intl` and `location` seem to be required

File: packages/admin/src/routes/entities/components/Breadcrumbs/Breadcrumbs.js [hunk 5]
```diff
@@ -43,7 +48,8 @@ const Breadcrumbs = ({breadcrumbsInfo}) => {
 
 Breadcrumbs.propTypes = {
   match: PropTypes.object,
-  breadcrumbsInfo: PropTypes.array
+  breadcrumbsInfo: PropTypes.array,
```
Comment (rzueger) [inline packages/admin/src/routes/entities/components/Breadcrumbs/Breadcrumbs.js:51]: maybe arrayof with shape?

File: packages/admin/src/routes/entities/components/Action/actions/CashManagementImport.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+import {goBack} from '../../../../../utils/routing'
+
+const CashManagement = props => {
+  const entityBaseUrl = goBack(props.match.url, 2)
+  return <Redirect to={entityBaseUrl + '/Cash_management_import/list?formName=Cash_management_import_action'}/>
+}
+
+CashManagement.propTypes = {
+  match: PropTypes.shape({
```
Comment (rzueger) [inline packages/admin/src/routes/entities/components/Action/actions/CashManagementImport.js:13]: match is required

File: packages/admin/src/routes/entities/components/Action/actions/ResourceScheduler.js [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+  const entityBaseUrl = goBack(match.url, 2)
+  return <ResourceSchedulerApp
+    onEventClick={({model, key}) => {
+      window.open(`${entityBaseUrl}/${model}/${key}`, '_blank')
+    }}
+  />
+}
+
+ResourceScheduler.propTypes = {
+  match: PropTypes.shape({
```
Comment (rzueger) [inline packages/admin/src/routes/entities/components/Action/actions/ResourceScheduler.js:17]: match is required

File: packages/admin/src/routes/entities/components/ActionView/ActionView.js (hunk 2 above)
Comment (MethodenMann) [reply packages/admin/src/routes/entities/components/ActionView/ActionView.js:24]: i thought you will work on this. I'll remove the comment anyway

File: packages/admin/src/routes/entities/components/ActionView/ActionView.js (hunk 3 above)
Comment (MethodenMann) [reply packages/admin/src/routes/entities/components/ActionView/ActionView.js:34]: 👍

**Pull request merged**